### DISCUSSION TITLE
Is it possible to preconfigure WiFi SSID and PSK?

### DISCUSSION THREAD

**Discussion by j12i** [johang/sd-card-images#153] [Q&A]
I wanted to try your image on a BananaPi M2-Magic. It doesn't have an ethernet port though, and after flashing an SD card I became aware that I wouldn't have a way to connect to the Linux system on the SBC after it booted.

Is there a way to write the SSID and PSK to the card from my PC? I looked for /etc/wpa_supplicant.conf on the card, but found nothing like that.

(thanks for your work of course johang)

**Comment by johang** [comment]
It's not enough to just configure SSID and PSK. You need to install `firmware-brcm80211` to get the AP6212 firmware. I would probably get one of those USB ethernet dongles and use that for initial setup.

**Comment by j12i** [comment]
Thank you.
